ui/raidboss/data/03-hw/dungeon/xelphatol.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+// Xelphatol
+[{
+  zoneId: ZoneId.Xelphatol,
+  timelineFile: 'xelphatol.txt',
+  timelineTriggers: [
+    {
+      id: 'Xelphatol Short Burst',
+      regex: /Short Burst/,
+      beforeSeconds: 4,
+      condition: Conditions.caresAboutMagical(),
+      suppressSeconds: 5, // Timelines jump sometimes, so let's not be noisy.
+      response: Responses.tankBuster(),
+    },
+    {
+      // This might be better handled by collecting Swiftfeather,
+      // but a timeline trigger is easy and consistent.
+      id: 'Xelphatol On Low',
+      regex: /On Low/,
+      beforeSeconds: 5,
+      response: Responses.tankCleave(),
```

Comment:
This is technically a getBehind, right? I feel like I remember avoiding this on tank.

Reply:
It's avoidable, but the timing is tight, on the order of less than a second, and improper avoidance can kill melee. (Or healers who happen to be close by. Not that I've ever done that.)

Reply:
Fair enough! Maybe leave this as a comment there for anybody else who sees it?